Base/Python/slicer/translation.py

```diff
@@ -0,0 +1,27 @@
+import os
+import inspect
+
+
+def get_context(source_file):
+    """ get the translation context based on the source file name"""
```

Comment:
Thanks for the suggestions, updates will be made on the next commit.

By the way, where could I find documentation on coding style rules used by the `CI / Lint` so that to avoid making it failed on next commits ?

Thanks in advance !